ucrop/src/main/java/com/yalantis/ucrop/util/BitmapLoadUtils.java

```diff
@@ -82,10 +95,10 @@ public static int calculateInSampleSize(@NonNull BitmapFactory.Options options,
         int inSampleSize = 1;
 
         if (height > reqHeight || width > reqWidth) {
-            // Calculate the largest inSampleSize value that is a power of 2 and keeps both
+            // Calculate the largest inSampleSize value and keeps both
             // height and width lower or equal to the requested height and width.
             while ((height / inSampleSize) > reqHeight || (width / inSampleSize) > reqWidth) {
-                inSampleSize *= 2;
+                inSampleSize++;
```

Comment:
From developer.android.com
If set to a value > 1, requests the decoder to subsample the original image, returning a smaller image to save memory. The sample size is the number of pixels in either dimension that correspond to a single pixel in the decoded bitmap. For example, inSampleSize == 4 returns an image that is 1/4 the width/height of the original, and 1/16 the number of pixels. Any value <= 1 is treated the same as 1. **Note: the decoder uses a final value based on powers of 2, any other value will be rounded down to the nearest power of 2.**